google/cloud/completion_queue_test.cc

```diff
@@ -513,6 +642,83 @@ TEST(CompletionQueueTest, NoRunAsyncAfterShutdown) {
   EXPECT_EQ(std::future_status::timeout, f.wait_for(ms(0)));
 }
 
+class RunAsyncTest : public ::testing::TestWithParam<int> {};
+
+TEST_P(RunAsyncTest, Torture) {
```

Comment:
It appears that these torture tests never fail (no expectations).  Is there something we could do to check that they are executing as we expect?  Is the general test timeout sufficient to check that the `wait()` completes, or would something more specific be nice?  Or would these just be better off as benchmarks?

Reply:
I added some reasonable expectations, but you are right.  These are largely smoke tests, if they crash (or fail under a sanitizer) we have a problem, otherwise we do not know (but Dijkstra would say that is true of all tests).

I do not feel very strongly about these tests, if you think they do not add enough value, I am happy to remove them.

Reply:
I'm happy to keep them.